test/performance/README.md

```diff
@@ -111,13 +111,86 @@ To generate the sample workload for KubeStellar performance experiments proceed
     - 10 config maps containing a 2048-character random string
 
 
-
-
 5. Clean up the generated workload Kubernetes API objects from your cluster:
 
    ```bash
    $ cd $CL2_DIR
    $ ./cleanup.sh
    ```
 
-### Collect the creation & update timestamps for workload benchmark (OPTIONAL) (TO DO: WORK-IN-PROGRESS)
\ No newline at end of file
+
+### Collect the creation & update timestamps for workload benchmark (OPTIONAL)
+
+
+To collect the creation and status_update timestamps for the benchmark workload objects proceed as following:  
+
+1. Run the metrics collection script:
+
+   a) cd into the `test/performance` directory from your local copy of the KubeStellar repo, for example:
+
+   ```bash
+   $ cd .$HOME/kubestellar/test/performance
+   ```
+
+   b) Run the metrics collection script:
+
+   ```bash 
+   $ python3 metrics_collector.py <kubeconfig> <wds_ctx> <its_ctx> <wec_ctx> <num_ns> <output_dir> 
+   ```
+
+   For example:
+   
+   ```bash 
+   $ python3 metrics_collector.py $HOME/.kube/config wds1 its1 cluster1 2 $HOME/data
+   ```
+
+   Below is a detailed explanation of the input parameters:
+   - `kubeconfig`: path to the kubeconfig file, e.g., *$HOME/.kube/config*
+   - `wds_ctx`: name of the context for the target WDS, e.g., *"wds1"*
```

Comment:
In bash the dquotes around `wds1` are not needed; bash parsing rules will isolate the proper "word" ("word" is a technical term in bash syntax) with or without the dquotes. Also, squotes could be used instead of dquotes. In fact, there are many ways to write the same word.

- `wds1`
- `"wds1"`
- `'wds1'`
- `\wds1`
- `wd\s1`
- `"wds"1`
- `'wd'"s1"`
- and on and on

I think it is best to just write it in the plainest way possible, lest the reader mistakenly conclude that there is something special going on here.

Reply:
Fixed!